File: 01284.cpp
// 집 주소

#include <iostream>
#include <string>
using namespace std;

int main() {
	while (1) {
		string s; cin >> s;
		if (s == "0") break;

		int res = s.length() + 1;
		for (int i = 0; i < s.length(); i++) {
			if (s[i] == '1') res += 2;
			else if (s[i] == '0') res += 4;
			else res += 3;
		}

		cout << res << endl;
	}
}

File: 01284.py
# 집 주소

while True:
    s = input()
    if s == '0':
        break
        
    res = len(s) + 1
    for i in range(len(s)):
        if s[i] == '1':
            res += 2
        elif s[i] == '0':
            res += 4
        else:
            res += 3
            
    print(res)
